README.md

```diff
@@ -1,18 +1,34 @@
 p4c-behavioral
 ========
 
+***This repository is
+   deprecated. [p4c-behavioral](https://github.com/p4lang/p4c-behavioral) has
+   been replaced by the
+   [behavioral-model](https://github.com/p4lang/behavioral-model), aka
+   bmv2. Please refer to this
+   [README](https://github.com/p4lang/behavioral-model/blob/master/README.md)
+   for more information. We have stopped providing maintenance for this
+   repository and be advised that it may be removed altogether from Github after
+   31/12/2017. If you have concerns about transitioning to bmv2, please send an
+   email to p4-dev@lists.p4.org.***
+
 P4 compiler for the behavioral model (BM) target
 
-Pre-requisites:  
-Install p4-hlir (https://github.com/p4lang/p4-hlir)
+Pre-requisites: [p4-hlir](https://github.com/p4lang/p4-hlir)
 
```

Comment:
I think you may want to move the rest of the instructions into a separate file (INSTALL or similar) so that only that paragraph remains.